samcli/commands/remote_invoke/remote_invoke_context.py

```diff
@@ -0,0 +1,122 @@
+"""
+Context object used by `sam remote invoke` command
+"""
+import logging
+from typing import Optional, cast
+
+from samcli.commands.remote_invoke.exceptions import (
+    AmbiguousResourceForRemoteInvoke,
+    InvalidRemoteInvokeParameters,
+    NoResourceFoundForRemoteInvoke,
+    UnsupportedServiceForRemoteInvoke,
+)
+from samcli.lib.pipeline.bootstrap.resource import ARNParts
```

Comment:
Should this be refactored to a more common place?

Reply:
That is clear now :) Will move this class into common place